velox/common/base/Status.cpp

```diff
@@ -54,6 +54,11 @@ std::string_view toString(StatusCode code) {
   return ""; // no-op
 }
 
+Status::Status(StatusCode code) {
```

Comment:
now that we are supporting any error code without message, it may make sense to pull `code` out of the state_, to prevent this allocation. Could do in a follow up PR though.